general_superstaq/user_config_test.py

```diff
@@ -80,6 +80,20 @@ def test_ibmq_set_token(  # pylint: disable=missing-function-docstring
     assert service.ibmq_set_token("valid token") == "Your IBMQ account token has been updated"
 
 
+@mock.patch(
+    "general_superstaq.superstaq_client._SuperstaQClient.cq_set_token",
```

Comment:
should also work and mocks (slightly) less:
```suggestion
    "general_superstaq.superstaq_client._SuperstaQClient.post_request",
```

Reply:
In this case should we also change the mock for the `ibmq_set_token` test?

Reply:
ah hadn't noticed that, yeah that'd be great!